pkg/cover/html.go

```diff
@@ -345,6 +345,7 @@ func fileContents(file *file, lines [][]byte, haveProgs bool) string {
 	for i, ln := range lines {
 		start := 0
 		cover := append(lineCover[i+1], lineCoverChunk{End: backend.LineEnd})
+		buf.WriteString(fmt.Sprintf("%d\t", i))
```

Comment:
Line numbers are generally numbered from 1. At least that's why all my editors do. I think this needs to be `i+1`.

Also I tried to mock this and printing line numbers in loop above where we print number of programs. And printing in the loop above looks nicer... kinda more visually separated from the code and in bold. Also code selection works. What do you think about placing numbers there?

Reply:
Done